pkg/transaction-builder/pocket_transaction_builder.go

```diff
@@ -0,0 +1,152 @@
+package transactionbuilder
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"errors"
+	"math"
+	"math/big"
+
+	"github.com/pokt-foundation/pocket-go/pkg/provider"
+	"github.com/pokt-foundation/pocket-go/pkg/signer"
+	"github.com/pokt-network/pocket-core/app"
+	"github.com/pokt-network/pocket-core/crypto"
+	coreTypes "github.com/pokt-network/pocket-core/types"
+	"github.com/pokt-network/pocket-core/x/auth"
+	authTypes "github.com/pokt-network/pocket-core/x/auth/types"
+)
+
+var (
+	// ErrNoSigner error when no signer is provided
+	ErrNoSigner = errors.New("no signer provided")
+	// ErrNoProvider error when no provider is provided
+	ErrNoProvider = errors.New("no provider provided")
+	// ErrNoChainID error when no chain ID is provided
+	ErrNoChainID = errors.New("no chain id provided")
+	// ErrNoTxMsg error when no Tx Msg is provided
+	ErrNoTxMsg = errors.New("no tx msg provided")
+)
+
+// PocketTransactionBuilder represents implementation of transaction builder package
+type PocketTransactionBuilder struct {
+	provider provider.Provider
+	signer   signer.Signer
+}
+
+// NewPocketTransactionBuilder returns an instance of PocketTransactionBuilder
+func NewPocketTransactionBuilder(provider provider.Provider, signer signer.Signer) *PocketTransactionBuilder {
+	return &PocketTransactionBuilder{
+		provider: provider,
+		signer:   signer,
+	}
+}
+
+func getOptionalParams(options *TransactionOptions) (string, string, int64) {
+	memo := ""
+	coinDenom := Upokt
+	fee := int64(10000)
```

Comment:
what that number means?

Reply:
Seems like it's the chain base fee. Should prob be a constant up above.

Reply:
I'll add a comment, it's the default fee for transactions